loopchain/consensus/runner.py

```diff
@@ -54,7 +55,18 @@ def __init__(self,
             self.event_system, ChannelProperty().peer_address, self._block_factory, self._vote_factory
         )
 
+        self._loop = asyncio.get_event_loop()
+        last_block_height = 0
+        if self._block_manager.blockchain.last_block:
```

Comment:
How about adding an attribute self._blockchain = self._block_manager.blockchain for all scope in this class. I think blockchain is using frequently in this file. I suppose to be better readability of this class if you accept it.